packages/components/src/DateTimePickers/pickers/DatePicker/DatePicker.test.js

```diff
@@ -1,14 +1,572 @@
+					monthIndex: 6,
+				};
+
+				const wrapper = shallow(<DatePicker
+					calendar={calendar}
+					onSelect={() => {}}
+				/>);
+
+				const days = getDayNumbers(wrapper);
+				const copy = [...days];
+				days.sort(ascSorting);
+				expect(days).toEqual(copy);
+				expect(days[0]).toBe(1);
+				expect(days).toHaveLength(31);
+			});
+
+			it('should have contiguous numbers for february 2016', () => {
+				const calendar = {
+					year: 2016,
+					monthIndex: 1,
+				};
+
+				const wrapper = shallow(<DatePicker
+					calendar={calendar}
+					onSelect={() => {}}
+				/>);
+
+				const days = getDayNumbers(wrapper);
+				const copy = [...days];
+				days.sort(ascSorting);
+				expect(days).toEqual(copy);
+				expect(days[0]).toBe(1);
+				expect(days).toHaveLength(29);
+			});
+
+			it('should have contiguous numbers for february 2017', () => {
+				const calendar = {
+					year: 2017,
+					monthIndex: 1,
+				};
+
+				const wrapper = shallow(<DatePicker
+					calendar={calendar}
+					onSelect={() => {}}
+				/>);
+
+				const days = getDayNumbers(wrapper);
+				const copy = [...days];
+				days.sort(ascSorting);
+				expect(days).toEqual(copy);
+				expect(days[0]).toBe(1);
+				expect(days).toHaveLength(28);
+			});
+
+			it('should have contiguous numbers for november 2022', () => {
+				const calendar = {
+					year: 2022,
+					monthIndex: 10,
+				};
+
+				const wrapper = shallow(<DatePicker
+					calendar={calendar}
+					onSelect={() => {}}
+				/>);
+
+				const days = getDayNumbers(wrapper);
+				const copy = [...days];
+				days.sort(ascSorting);
+				expect(days).toEqual(copy);
+				expect(days[0]).toBe(1);
+				expect(days).toHaveLength(30);
+			});
+		});
+
+		describe('current date', () => {
+			it('should render specifically the current date', () => {
+				const calendar = {
+					year: 2018,
+					monthIndex: 5,
+				};
+
+				mockIsTodayWith(new Date(2018, 5, 13));
+
+				const wrapper = shallow(<DatePicker
+					calendar={calendar}
+					onSelect={() => {}}
+				/>);
+
+				const currentDayItems = wrapper
+					.find('.theme-calendar-body-row .theme-calendar-item DayPickerAction')
+					.filterWhere(item => item.prop('isToday'));
+
+				expect(currentDayItems).toHaveLength(1);
+				const item = currentDayItems.first();
+				expect(item.prop('label')).toBe('13');
+			});
+
+			it('should not render specifically a date if current date is out of displayed month', () => {
+				const calendar = {
+					year: 2018,
+					monthIndex: 5,
+				};
+
+				mockIsTodayWith(new Date(2018, 4, 13));
+
+				const wrapper = shallow(<DatePicker
+					calendar={calendar}
+					onSelect={() => {}}
+				/>);
+
+				const currentDayItems = wrapper
+					.find('.theme-calendar-body-row .theme-calendar-item DayPickerAction')
+					.filterWhere(item => item.prop('isToday'));
+
+				expect(currentDayItems).toHaveLength(0);
+			});
+
+			it.only('should have updated the current today day if has changed between two renders', () => {
```

Comment:
You tested that :D
but what is this `it.only` ?

Reply:
oups, already commit a fix to remove the '.only'.
✔